featuretools/feature_base/features_deserializer.py

```diff
@@ -15,20 +19,23 @@
 from featuretools.utils.gen_utils import check_schema_version
 
 
-def load_features(features):
-    """Loads the features from a filepath, an open file, or a JSON formatted string.
+def load_features(features, profile_name=None):
+    """Loads the features from a filepath, S3 path, URL, an open file, or a JSON formatted string.
 
     Args:
         features (str or :class:`.FileObject`): The location of where features has
         been saved which this must include the name of the file, or a JSON formatted
         string, or a readable file handle where the features have been saved.
 
+        profile_name (str): The AWS profile specified to write to S3.
+                            Default: None
+
     Returns:
         features (list[:class:`.FeatureBase`]): Feature definitions list.
 
     Note:
-        Features saved in one version of Featuretools are not guaranteed to work in another.
-        After upgrading Featuretools, features may need to be generated again.
+        Features saved in one version of Featuretools or python are not guaranteed to work in another.
```

Comment:
Can you create an issue about this?  That feature lists generated by python 3.5 and python 3.6+ have different orderings?

Reply:
Added issue: https://github.com/Featuretools/featuretools/issues/687